ros2_batch_job/__main__.py

```diff
@@ -49,7 +49,15 @@
     'EmPy',
     'coverage',
     'flake8',
+    'flake8-blind-except',
+    'flake8-builtins',
+    'flake8-class-newline',
+    'flake8-commas',
```

Comment:
We talked about the list of plugins in todays meeting and will keep all but this one.